### ISSUE TITLE
Autoslim on object detection?

### ISSUE THREAD

**Issue by twmht** [open-mmlab/mmrazor#89]
Hi,

I have tried to prune object detection model with autoslim.

But the accuracy is always low, even the training loss is normal.

I have checked the original repo (https://github.com/JiahuiYu/slimmable_networks/tree/detection) has done the detection. But there are some details they don't mentioned.

For example, do we need to pretrain slimmable model in ImageNet and then transfer to object detection?

There are also the related work (https://arxiv.org/abs/2103.13258)  they use the pretrained slimmable model in imagenet. But I am not sure if this is the requirement.

Any idea?

**Comment by hunto** [comment]
Hi,
The pre-training of autoslim supernet on ImageNet **is required** when pruning on object detection, as the convergency of detection is highly influenced by the pre-training performance.

BTW, if you directly evaluate the accuracies of subnets using a normally-trained checkpoint (e.g., ckpt provided by torchvision), the results would be poor. That's why we need to load a autoslim-pretrained checkpoint on detection.

**Comment by twmht** [comment]
@hunto
